src/services/kadenaService.js

```diff
@@ -57,54 +70,93 @@ async function getKadenaHeight(ip) {
     const agent = new https.Agent({
       rejectUnauthorized: false,
     });
-    const kadenaData = await axios.get(`https://${ip}:30004/chainweb/0.0/mainnet01/cut`, { httpsAgent: agent, timeout: 13456 });
+    const kadenaData = await http.get(`https://${ip}:30004/chainweb/0.0/mainnet01/cut`, { httpsAgent: agent, timeout: 13456 });
     return kadenaData.data.height;
   } catch (e) {
-    // log.error(e);
+    log.error(`getKadenaHeight of IP ${ip} error`);
     return -1;
   }
 }
 
 async function getFluxNodeTier(ip) {
   try {
-    const fluxnodeList = await axios.get(`http://${ip}:16127/daemon/getzelnodestatus`, axiosConfig);
+    const fluxnodeList = await http.get(`http://${ip}:16127/daemon/getzelnodestatus`, axiosConfig);
     if (fluxnodeList.data.status === 'success') {
       return fluxnodeList.data.data.tier;
     }
-    return null;
+    return false;
   } catch (e) {
-    // log.error(e);
-    return null;
+    log.error(`getFluxNodeTier of IP ${ip} error`);
+    return false;
   }
 }
 
 async function getFluxNodeZelID(ip) {
   try {
-    const fluxnodeList = await axios.get(`http://${ip}:16127/flux/zelid`, axiosConfig);
+    const fluxnodeList = await http.get(`http://${ip}:16127/flux/zelid`, axiosConfig);
     if (fluxnodeList.data.status === 'success') {
       return fluxnodeList.data.data;
     }
-    return null;
+    return false;
   } catch (e) {
-    // log.error(e);
-    return null;
+    log.error(`getFluxNodeZelID of IP ${ip} error`);
+    return false;
   }
 }
 
 async function getKadenaVerison(ip) {
   try {
-    const appData = await axios.get(`http://${ip}:16127/apps/installedapps/KadenaChainWebNode`, axiosConfig);
+    const appData = await http.get(`http://${ip}:16127/apps/installedapps/KadenaChainWebNode`, axiosConfig);
     if (appData.data.status === 'success') {
       return appData.data.data[0].hash;
     }
-    return null;
+    return false;
   } catch (e) {
-    // log.error(e);
-    return null;
+    log.error(`getKadenaVerison of IP ${ip} error`);
+    return false;
   }
 }
 
+async function processKdaNode(kdaNode, currentRoundTime, synced) {
```

Comment:
This should still store a value if some requests fails. 
A bad call for zelid/tier or something shall still store the node into database as before